nilearn/plotting/displays.py

```diff
@@ -1203,6 +1203,242 @@ def draw_cross(self, cut_coords=None, **kwargs):
+        if cut_coords is None:
+            if img is None or img is False:
+                cut_coords = (0, 0, 0)
+            else:
+                cut_coords = find_xyz_cut_coords(
+                    img, activation_threshold=threshold)
+            cut_coords = [cut_coords['xyz'.find(c)]
+                          for c in sorted(self._cut_displayed)]
+        return cut_coords
+
+    def _init_axes(self, **kwargs):
+        cut_coords = self.cut_coords
+        if len(cut_coords) != len(self._cut_displayed):
+            raise ValueError('The number cut_coords passed does not'
+                             ' match the display_mode')
+        x0, y0, x1, y1 = self.rect
+        facecolor = 'k' if self._black_bg else 'w'
+        # Create our axes:
+        self.axes = dict()
+        for index, direction in enumerate(self._cut_displayed):
+            fh = self.frame_axes.get_figure()           
+            if (index == 0):
+                coord1 = x1 - x0
+                coord2 = 0.5 * (y1-y0) + y0
+                coord3 = 0.5 * (x1 - x0) + x0
+                coord4 = y1-y0
+
+            if (index == 1):
+                coord1 = 0.5 * (x1 - x0) + x0
+                coord2 = 0.5 * (y1-y0) + y0
+                coord3 = x1 -x0
+                coord4 = y1-y0
+
+            if (index == 2):
+                coord1 = x1 - x0
+                coord2 = y1 -y0
+                coord3 = 0.5 * (x1 - x0) + x0
+                coord4 = 0.5 * (y1-y0) + y0
+
+            
+            ax = fh.add_axes([coord1,coord2,coord3,coord4], aspect='equal')
+
+            if LooseVersion(matplotlib.__version__) >= LooseVersion("1.6"):
+                ax.set_facecolor(facecolor)
+            else:
+                ax.set_axis_bgcolor(facecolor)
+
+            ax.axis('off')
+            coord = self.cut_coords[
+                sorted(self._cut_displayed).index(direction)]
+            display_ax = self._axes_class(ax, direction, coord, **kwargs)
+            self.axes[direction] = display_ax
+            ax.set_axes_locator(self._locator)
+        
+        if self._black_bg:
+            for ax in self.axes.values():
+                ax.ax.imshow(np.zeros((2, 2, 3)),
+                             extent=[-5000, 5000, -5000, 5000],
+                             zorder=-500, aspect='equal')
+
+            # To have a black background in PDF, we need to create a
+            # patch in black for the background
+            self.frame_axes.imshow(np.zeros((2, 2, 3)),
+                                   extent=[-5000, 5000, -5000, 5000],
+                                   zorder=-500, aspect='auto')
+            self.frame_axes.set_zorder(-1000)
+
+    def _locator(self, axes, renderer):
+        """ The locator function used by matplotlib to position axes.
+            Here we put the logic used to adjust the size of the axes.
+        """
+
+        x0, y0, x1, y1 = self.rect
+        width_dict = dict()
+        height_dict = dict()
```

Comment:
`width_dict` & `height_dict` are not good variable names. 

1. It is bad practice to include the datatype (dict) in the variable name.
2. width & height are very generic names. The variable names should be self-documenting, for example: `width_of _<something>`, `height_of _<something>`

Reply:
We tried to use the same names or logic for variable names as in the OrthoSlicer Subclass wherever possible for consistency. Both `width_dict` and `height_dict` are used in the OrthoSlicer Subclass similarly. Would you prefer we change them only for the new class TiledSlicer now, or keep them and maybe open another issue to change variable names for OrthoSlicer as well?